packages/foreman/foreman-proxy/foreman-proxy.spec

```diff
@@ -183,22 +178,13 @@ if [ $1 == 2 ]; then
   popd >/dev/null
 fi
 
-if [ $1 -eq 1 ]; then
-  /bin/systemctl daemon-reload >/dev/null 2>&1 || :
-fi
+%systemd_post %{name}.service
```

Comment:
This generates a slightly different code that might actually be more appropriate:
```
if [ $1 -eq 1 ] ; then 
        # Initial installation 
        systemctl preset foreman-proxy.service >/dev/null 2>&1 || : 
fi
```